app/javascript/readingList/__tests__/__snapshots__/collectionForm.test.jsx.snap

```diff
@@ -1,51 +1,47 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`<CollectionForm /> renders properly 1`] = `
-<div
-  class="articleformcontainer"
+<section
+  class="articleform"
 >
-  <section
-    class="articleform"
+  <form
+    class="articleform__form"
   >
-    <form
-      class="articleform__form"
+    <h1>
+      Add your new collection
+    </h1>
+    <input
+      class="articleform__title"
+      name="title"
+      onChange={[Function]}
+      placeholder="Collection Name..."
+      required={true}
+      type="text"
+    />
+    <div
+      class="articleform__tagswrapper"
     >
-      <h1>
-        Add your new collection
-      </h1>
       <input
-        class="articleform__title"
-        name="title"
-        onChange={[Function]}
-        placeholder="Title..."
-        required={true}
+        autoComplete="off"
+        class="articleform__tags"
+        id="tag-input"
+        onBlur={[Function]}
+        onFocus={[Function]}
+        onInput={[Function]}
+        onKeyDown={[Function]}
+        placeholder="4 tags max, comma separated, no spaces or special characters"
```

Comment:
We could change this if needed. Is there anything that pops up if we click on more than 4 tags?